make-options.json

```diff
@@ -245,6 +246,7 @@
         "FtpUploadV2",
         "MSBuildV1",
         "NodeToolV0",
+        "NotationV0",
```

Comment:
You don't need to add the task in "Node20" section. Such sections (Node20, Node16_225 etc) are only for generated tasks

Reply:
Removed.